packages/astro/src/env/runtime-constants.ts

```diff
@@ -0,0 +1 @@
+export const ENV_SYMBOL = Symbol.for('astro:env/dev');
```

Comment:
Can't be in `constants.ts` or other constants would be imported in the setup chunk, and Cloudflare breaks

Reply:
Insane 🥲 I wonder if we should also inline the symbol directly instead of sharing the same file. That way we don't need a single file for this case. But I'll leave it to you to decide.

Reply:
I prefer a file, I don't like magic things